test/regression/expected/transaction_errors.out

```diff
@@ -1,6 +1,8 @@
 CREATE TABLE foo AS SELECT 'bar'::text AS t;
 BEGIN; SET duckdb.force_execution = true; SELECT t::integer AS t1 FROM foo; ROLLBACK;
-ERROR:  invalid input syntax for type integer: "bar"
+ERROR:  (PGDuckDB/Duckdb_ExecCustomScan) Conversion Error: Could not convert string 'bar' to INT32
```

Comment:
Previously this was being executed in Postgres, now it's executed in duckdb again. We still don't crash though.